src/pages/restaurantPage/RestaurantPage.js

```diff
@@ -0,0 +1,71 @@
+//Libraries
```

Comment:
since the directory name is page, you don't need to repeat page inside component name, you can call this Restaurant itself